src/Donations/Models/Donation.php

```diff
@@ -28,33 +28,34 @@
  * @since 2.20.0 update amount type, fee recovered, and exchange rate
  * @since 2.19.6
  *
- * @property int $id
- * @property int $formId
- * @property string $formTitle
- * @property DateTime $createdAt
- * @property DateTime $updatedAt
+ * @property int            $id
+ * @property int            $formId
+ * @property string         $formTitle
+ * @property DateTime       $createdAt
+ * @property DateTime       $updatedAt
  * @property DonationStatus $status
- * @property DonationMode $mode
- * @property DonationType $type
- * @property Money $amount amount charged to the gateway
- * @property Money $feeAmountRecovered
- * @property string $exchangeRate
- * @property string $gatewayId
- * @property int $donorId
- * @property string $firstName
- * @property string $lastName
- * @property string $email
- * @property int $subscriptionId
+ * @property DonationMode   $mode
+ * @property DonationType   $type
+ * @property Money          $amount amount charged to the gateway
+ * @property Money          $feeAmountRecovered
+ * @property string         $exchangeRate
+ * @property string         $gatewayId
+ * @property int            $donorId
+ * @property string         $firstName
+ * @property string         $lastName
+ * @property string         $email
+ * @property int            $subscriptionId
  * @property BillingAddress $billingAddress
- * @property string $purchaseKey
- * @property string $donorIp
- * @property bool $anonymous
- * @property string $levelId
- * @property string $gatewayTransactionId
- * @property Donor $donor
- * @property Subscription $subscription
+ * @property string         $purchaseKey
+ * @property string         $donorIp
+ * @property bool           $anonymous
+ * @property string         $levelId
+ * @property string         $gatewayTransactionId
+ * @property Donor          $donor
+ * @property Subscription   $subscription
```

Comment:
We don't worry about spacing out the properties like this, as it can make for gnarly reformatting when a new property is introduced. Please update your formatting to not align doc properties.

Reply:
Resolved 2ec055b6da103a957f6a73d78a6603be5675e6bc